network/topics/params/topic_score.go

```diff
@@ -10,8 +10,9 @@ import (
 
 const (
 	// Network Topology
-	gossipSubD          = 8
-	minActiveValidators = 200
+	gossipSubD                = 8
+	minActiveValidators       = 200
+	msgsPerSecondPerValidator = 0.05
```

Comment:
how did you arrive to this number?

Reply:
0.05 comes from the estimation of the number of messages received in the sheets.